app/controllers/incidents_controller.rb

```diff
@@ -13,11 +13,13 @@ class IncidentsController < ApplicationController
 
   def index
     base_model = Incident.includes(:competitions, :incident_tags)
-    if current_user&.can_manage_incidents?
-      @incidents = base_model.all
-    else
-      @incidents = base_model.resolved
-    end
+    @incidents = if current_user&.can_manage_incidents? # WRC members see all
+                   base_model.all
+                 elsif current_user&.can_view_delegate_matters? # Staff see staff + public
+                   base_model.staff_visible
```

Comment:
The name of the scope (aka `staff_visible`) implies that "if you're Staff in any capacity, you should be able to see this incident". The check you're performing however is whether the current user can access _Delegate Matters_, which is distinct from being _staff_.

For example, Software Team members are staff but they cannot see Delegate matters. Same goes for WSOT or WMCT for example.

So either you change the check to something that relates to the user being _staff_, or you should rename the visibility scope for the incidents to something like `delegate_visible` (name tentative, feel free to come up with something better)